src/Jackett.Common/Definitions/thepiratebay.yml

```
@@ -140,14 +140,20 @@ search:
   keywordsfilters:
     # remove it's #8829
     - name: re_replace
-      args: ["(?i)(it's\\s)", ""]
-    # replace simplified chinese as this confuses TPB search engine #7291
+      args: ["(?i)\\bit's\\b", ""]
+    # replace simplified Chinese as this confuses TPB search engine #7291
     - name: re_replace
       args: ["([\\p{IsCJKUnifiedIdeographs}\\W]+)", "."]
+    # search for both S01 and Season 01
+    - name: re_replace
+      args: ["(?i)\\b(S0?(\\d{1}))\\b", "$1|\"Season.0$2\""]
+    # search for both S10 and Season 10
+    - name: re_replace
+      args: ["(?i)\\b(S(\\d{2,3}))\\b", "$1|\"Season.$2\""]
```

```suggestion
    # search for both S01 and Season 01
    - name: re_replace
      args: ["(?i)\\b(S(\\d{2,3}))\\b", "$1|\"Season.$2\""]
```

Ignore the earlier suggestion. This?